tasks/cephfs/test_failover.py

```diff
@@ -570,3 +570,57 @@ def set_standby_for(leader_fs, follower_id):
         fs_b.wait_for_daemons()
         self.assertEqual(set(fs_a.get_active_names()), {mds_a, mds_b})
         self.assertEqual(set(fs_b.get_active_names()), {mds_c, mds_d})
+
+    def test_standby_for_invalid_fscid(self):
```

Comment:
Please separate this new test from the addition of rsync, they should be separate pull requests.